tests/conftest.py

```diff
@@ -35,4 +35,11 @@ def get_content_dict(response):
     return content_dict
 
 
+def assert_jwt_account_response(content_dict):
+    assert(all([key in content_dict for key in
+                ["jwt", "account"]]))
+    assert(all([key in content_dict["jwt"] for key in
+                ["access_token", "refresh_token", "token_type"]]))
+
+
 TEST_EMAIL_DOMAIN = "@testing.fastsurvey.io"
```

Comment:
As test email I use `test@fastsurvey.io`, if I need more than one test email, I use `test+0@fastsurvey.io`, `test+1@fastsurvey.io`, ...

Easier to manage as the letters you're using now, and you would be sending from the correct domain (which you should please do). As a plus, when we migrate to GMail, you'll receive all the emails sent to these addresses on the single `test@fastsurvey.io` inbox due to GMail's +Aliases.

Reply:
Oh, that is nice :) Thanks! Will change that!